app/jobs/export_tables_to_cloud_storage_job.rb

```diff
@@ -0,0 +1,9 @@
+require 'export_tables_to_cloud_storage'
+
+class ExportTablesToCloudStorageJob < ApplicationJob
+  queue_as :export_tables
+
+  def perform
+    ExportTablesToCloudStorage.new.run!
```

Comment:
This is also defined in `def perform` in the `ImportCSVToBigQueryJob`. I think this was done so that the actions run in sequence.

I think this file is no longer needed.